README.md

```diff
@@ -5,11 +5,15 @@ This project contains an example server built with the Express framework written
 ## Quick Start
 
 - Use the Node version found in the .nvmrc file. This project uses `v21.1.0`.
-- Install the dependencies by running `pnpm install`. Here we use pnpm but you can install with a manager of your choice.
-- Rename [`config.example.json`](https://github.com/sednasystems/sedna-canvas-example/blob/main/config.example.json) in the root of the project to `config.json`.
-- To start the server in development, run `pnpm dev`. `npm dev` or `yarn dev` if you are using another package manager. This will start the Express server and `nodemon` will listen to any changes during development.
+- `pnpm install`
+- `mv config.example.json config.json`
```

Comment:
If we do this, won't this immediately seem like a change to the files committed to git? Should it maybe do a `cp` instead of a `mv`, and we should gitignore `config.json`? (I don't have enough context to know for sure if that makes sense here, but it would be good to avoid these setup instructions immediately giving you a git change.

Reply:
yes good catch I'm not sure why I wrote that - updated